src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1295,6 +1291,14 @@
     <Content Include="sharpdx_direct3d11_effects_x86.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="ViewModels\Watch3D\shaderSource\helix_shader_defs\COMMON\Common.hlsl" />
```

Comment:
@aparajit-pratap does this copy the files to the bin directory? We do not want that since the we'll distribute this with our binaries.

Reply:
this leads me to believe we should change this back to None 
https://stackoverflow.com/questions/1060870/in-a-csproj-file-what-is-none-include-for

Reply:
No, this doesn't.

Reply:
Changed them to `None`.